pos_loyalty/models/pos_config.py

```diff
@@ -1,6 +1,6 @@
-# -*- coding: utf-8 -*-
 # Copyright 2004-2010 OpenERP SA
 # Copyright 2017 RGB Consulting S.L. (https://www.rgbconsulting.com)
+# Copyright 2018 Lambda IS DOOEL <https://www.lambda-is.com>
```

Comment:
You added copyright in files you did'nt changed. (done many times).

Reply:
encoding declaration `# -*- coding: utf-8 -*-` line was removed on all .py files. Not sure if that constitutes as a change.

Reply:
Sorry but I don't agree with this argument. I think overall **all** changes should be considered, even cosmetic ones, since they increase the maintainability of the file/project. Otherwise there is no point in doing them.

Although this is a really trivial case so I'll just remove the copyright.